tests/operators/inverse_mass_variable_coefficient.output

```diff
@@ -0,0 +1,306 @@
+  element_type     = Simplex
+  dim              = 2
+  n_components     = 1
+  is_dg            = 1
+  RT_on_hypercube  = 0
+  InverseMassType  = ElementwiseKrylovSolver
+  Number of degrees of freedom: 384
+
+  ||ref||_infty = 0.999994
+  &src == &dst in InverseMassOperator: || vec1 - vec2||_infty = 0
+  consider_inverse_coefficient = 0   : || vec  - ref ||_infty = 3.21965e-15
+
+  ||ref||_infty = 0.99797
+  &src == &dst in InverseMassOperator: || vec1 - vec2||_infty = 0
+  consider_inverse_coefficient = 1   : || vec  - ref ||_infty = 3.44169e-15
+
+
+  element_type     = Simplex
+  dim              = 2
+  n_components     = 2
+  is_dg            = 1
+  RT_on_hypercube  = 0
+  InverseMassType  = ElementwiseKrylovSolver
+  Number of degrees of freedom: 768
+
+  ||ref||_infty = 0.999241
+  &src == &dst in InverseMassOperator: || vec1 - vec2||_infty = 0
+  consider_inverse_coefficient = 0   : || vec  - ref ||_infty = 3.77476e-15
+
+  ||ref||_infty = 0.998142
+  &src == &dst in InverseMassOperator: || vec1 - vec2||_infty = 0
+  consider_inverse_coefficient = 1   : || vec  - ref ||_infty = 3.9968e-15
+
+
+  element_type     = Simplex
+  dim              = 3
+  n_components     = 1
+  is_dg            = 1
+  RT_on_hypercube  = 0
+  InverseMassType  = ElementwiseKrylovSolver
+  Number of degrees of freedom: 3840
+
+  ||ref||_infty = 0.99966
+  &src == &dst in InverseMassOperator: || vec1 - vec2||_infty = 0
+  consider_inverse_coefficient = 0   : || vec  - ref ||_infty = 3.60822e-15
+
+  ||ref||_infty = 0.999696
+  &src == &dst in InverseMassOperator: || vec1 - vec2||_infty = 0
+  consider_inverse_coefficient = 1   : || vec  - ref ||_infty = 3.55271e-15
+
+
+  element_type     = Simplex
+  dim              = 3
+  n_components     = 3
+  is_dg            = 1
+  RT_on_hypercube  = 0
+  InverseMassType  = ElementwiseKrylovSolver
+  Number of degrees of freedom: 11520
+
+  ||ref||_infty = 0.999992
+  &src == &dst in InverseMassOperator: || vec1 - vec2||_infty = 0
+  consider_inverse_coefficient = 0   : || vec  - ref ||_infty = 4.73233e-15
+
+  ||ref||_infty = 0.999968
+  &src == &dst in InverseMassOperator: || vec1 - vec2||_infty = 0
+  consider_inverse_coefficient = 1   : || vec  - ref ||_infty = 4.85723e-15
+
+
+  element_type     = Hypercube
+  dim              = 2
+  n_components     = 1
+  is_dg            = 1
+  RT_on_hypercube  = 0
+  InverseMassType  = MatrixfreeOperator
+  Number of degrees of freedom: 72
+
+  ||ref||_infty = 0.998761
+  &src == &dst in InverseMassOperator: || vec1 - vec2||_infty = 0
+  consider_inverse_coefficient = 0   : || vec  - ref ||_infty = 1.02002e-15
+
+  ||ref||_infty = 0.968522
+  &src == &dst in InverseMassOperator: || vec1 - vec2||_infty = 0
+  consider_inverse_coefficient = 1   : || vec  - ref ||_infty = 1.44329e-15
+
+
```

Comment:
Here is the problem I found:
for the `MatrixFreeOperator`, using more than on component leads to a wrong result.
One component (which we also used for the scalar pressure mass operator with variable coefficient in the preconditioner) _does work_!

Reply:
yes I think this is easier :) thank you for your comments